src/fragments/lib/auth/js/set-up-auth.mdx

```diff
@@ -269,11 +265,11 @@ If your existing UserPool client has a required attribute that is NOT set to mut
 </Block>
 </BlockSwitcher>
 
-**Recap:** You now have set up and configured your backend resources for Amplify authentication and are ready to start working on your front-end application.
+You now have set up and configured your backend resources for Amplify authentication and are ready to start working on your frontend application.
 
 ## Determine your Auth integration path
 
-Before you implement Auth on your front-end application, you will want to evaluate the options of using the Authenticator connected UI component or using the Amplify Library APIs directly. We recommend using the Authenticator since it creates a sign in and registration experience for your with few lines of code.
+Before you implement Auth on your frontend application, you will want to evaluate the options of using the Authenticator connected UI component or using the Amplify Library APIs directly. We recommend using the Authenticator since it creates a sign in and registration experience for your app with a few lines of code.
```

Comment:
Do we introduce in this section a link to the new Enable sign-up, sign-in, sign-out guide for those who want to use libraries?

Reply:
@abdallahshaban557 Forgot to add here - I think what we may be able to sign-post more is when I switch to the other guide and use libraries for this set up. Maybe as part of the recap CTA on line 286 - if they want to change course follow the link then? open to ideas!

Reply:
hey - not sure I quite understand this here. Maybe we can chat about it and move forward from there.

Reply:
Updated with the link here to the current sign-in page but we may want to look at naming conventions there and forwarding to an updated page URL since the current page is called emailpassword.mdx

Reply:
Before you implement Auth on your frontend application, you will want to evaluate the options of using the Authenticator connected UI component or using the Amplify Library APIs directly. We recommend using the Authenticator since it creates a **customizable** sign in and registration experience for your app with a few lines of code.

Reply:
Updated